lexicons/tools/ozone/set/add.json

```diff
@@ -0,0 +1,31 @@
+{
+  "lexicon": 1,
+  "id": "tools.ozone.sets.add",
+  "defs": {
+    "main": {
+      "type": "procedure",
+      "description": "Add values to a specific set",
+      "input": {
+        "encoding": "application/json",
+        "schema": {
+          "type": "object",
+          "required": ["name", "values"],
+          "properties": {
+            "name": {
+              "type": "string",
+              "description": "Name of the set to add values to"
+            },
+            "values": {
+              "type": "array",
+              "minLength": 1,
+              "items": {
+                "type": "string"
+              },
```

Comment:
Do we want any limits on set item sizes?

Reply:
that is a good point. I think something like 2048 bytes? or 8kbyte to align with the `uri` string format? don't need a grapheme limit for this one.